edb/testbase/server.py

```diff
@@ -483,6 +484,11 @@ def _assert_generic_shape(path, data, shape):
                     self.fail(
                         f'{message}: not isclose({data}, {shape}) '
                         f'{_format_path(path)}')
+            elif isinstance(shape, decimal.Decimal):
```

Comment:
Why not simply add to the isinstance tuple below?

Reply:
Done.